.github/workflows/ci.yml

```diff
@@ -4,7 +4,9 @@ name: CI
 env:
   MAFIA_BUILD: "1938"
   MAFIA_VERSION: "20430"
-  SCRIPT_NAME: "autoscend.ash"
+  SCRIPT_NAME1: "autoscend.ash"
+  SCRIPT_NAME2: "auto_pre_adv.ash"
+  SCRIPT_NAME3: "auto_post_adv.ash"
```

Comment:
actually important. it errors if you just try to put "auto_pre_adv.ash" into the command instead of SCRIPT_NAME2